index.html

```diff
@@ -566,8 +587,134 @@ <h1>Trust Model</h1>
 
     <section>
       <h1>Basic Concepts</h1>
+
+      <section>
+        <h2>Types</h2>
+
+        <p>
+Type information may be expressed via the <a>type</a> property:
+        </p>
+        <dl>
+          <dt><dfn>type</dfn></dt>
+          <dd>
+The value of this property MUST be one or more URIs.
+If more than one URI is provided, the array MUST be interpreted as a set of
```

Comment:
I recommend changing "If more than one URI ..." to

"If more than one URI is provided, the URIs must be understood be an unordered set. If a syntax, such as JSON-LD, uses an array to represent the set of URIs, the order of the values is not significant. Note that syntax..."

Reply:
Fixed in 782accd34b749f362518646d8d392fdd5eee1b23.